gp-includes/things/translation.php

```diff
@@ -81,6 +81,11 @@ public function fix_translation( $translation ) {
 
 	public function restrict_fields( $translation ) {
 		$translation->translation_0_should_not_be( 'empty_string' );
+		$translation->translation_1_should_not_be( 'empty_string' );
+		$translation->translation_2_should_not_be( 'empty_string' );
+		$translation->translation_3_should_not_be( 'empty_string' );
+		$translation->translation_4_should_not_be( 'empty_string' );
+		$translation->translation_5_should_not_be( 'empty_string' );
```

Comment:
@toolstack No, I've tested it with 1 (zh_CN) and 3 (ru_RU) plural forms. If a locale only uses 3 plurals, the other 3 are null, not an empty string, so the validation passes.